client/scripts/views/pages/subscriptions/chainEventSettings.ts

```diff
@@ -72,12 +71,48 @@ const EventSubscriptionRow: m.Component<IEventSubscriptionRowAttrs, {}> = {
   }
 };
 
+const IndividualEventSubscriptions: m.Component<{}, {
+  chain: string;
+  eventKinds: IChainEventKind[];
+  allSupportedChains: string[];
+  isSubscribedAll: boolean;
+  isEmailAll: boolean;
+}> = {
+  view: (vnode) => {
+    let titler;
+    if (vnode.state.chain === ChainNetwork.Edgeware || vnode.state.chain === 'edgeware-local') {
```

Comment:
since we are checking against the chain id, it should just be `edgeware` and `edgeware-local`. if we want to check against ChainNetwork (which is slightly preferable), we should look that up instead.